wikimedia/wikipedia-ios
# Talk pages - cell design review tweaks

**Phabricator:** 
https://phabricator.wikimedia.org/T310279#8307471

### Notes
This PR adds a time to the timestamp label as well as adjust the colors of the reply depth indicators. To match Android, we're going with showing the timestamp in the user's local timezone. Note that this does mean it may be a different time than what is displayed in their signature, you can see the comment thread starting at https://phabricator.wikimedia.org/T310279#8330402 for more details.

For the format - I'm trying to match the mocks as close as I can, but used `setLocalizedDateFormatFromTemplate` to allow the system to adjust it to locale conventions. Here's an example of how it looks on an EN Wiki talk page vs. a DE Wiki talk page:

EN:
![IMG_0839](https://user-images.githubusercontent.com/3620196/197253337-31098184-02a9-4140-82c7-20b46a67ff9f.PNG)
DE:
![IMG_0840](https://user-images.githubusercontent.com/3620196/197253333-a218636c-4b5a-4a77-ab78-9e99ca715de6.PNG)

**Test Steps:**

1. Confirm the time displays in your device timezone local time. Confirm the format changes depending on the Wiki you are looking at. Note: RTL layout changes have not been applied yet. That will be in a followup PR.
2. Confirm reply depth indicators have changed according to Phab comment.
